ceph.spec.in

```diff
@@ -79,6 +79,10 @@ BuildRequires:	python-requests
 %if ( 0%{?rhel} > 0 && 0%{?rhel} < 7 ) || ( 0%{?centos} > 0 && 0%{?centos} < 7 )
 BuildRequires:	python-sphinx10
 %endif
+%if 0%{?fedora} || 0%{defined suse_version}
```

Comment:
I dont currently have the ability to compile centos/rhel 7 (EPEL is not included in OBS) so I have to take guidance if you want this in RHEL.